courses/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.0.1 on 2022-06-17 20:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
The model was removed in this branch so this migration file should also be removed.